kernel/serial_port/src/lib.rs

```diff
@@ -384,7 +384,10 @@ static INTERRUPT_ACTION_COM1_COM3: Once<Box<dyn Fn() + Send + Sync>> = Once::new
 static INTERRUPT_ACTION_COM2_COM4: Once<Box<dyn Fn() + Send + Sync>> = Once::new();
 
 
-// Cross-platform interrupt handler for COM1 and COM3 (IRQ 0x24 on x86_64).
+// Cross-platform interrupt handler for the primary serial port.
+//
+// * On x86_64, this is IRQ 0x24, used for COM1 and COM3 serial ports.
+// * On aarch64, we also use interrupt 0x24, used for the 
```

Comment:
On aarch64, we use interrupt 33 / 0x21 for serial port reception (we have no choice).
By the way, I think this comment is unfinised.

Reply:
ah yes, thanks!